utils/filter.c

```diff
@@ -45,8 +45,10 @@ static void print_trigger(struct uftrace_trigger *tr)
 	if (tr->flags & TRIGGER_FL_FILTER) {
 		if (tr->fmode == FILTER_MODE_IN)
 			pr_dbg("\ttrigger: filter IN\n");
-		else
+		else if (tr->fmode == FILTER_MODE_OUT)
 			pr_dbg("\ttrigger: filter OUT\n");
+		else
+			pr_dbg("\ttrigger: filter NONE\n");
```

Comment:
I think it should `tr->fmode` set when it has `TRIGGER_FL_FILTER` flag.

Reply:
Sorry, what do you mean? That the `NONE` value isn't valid when `TRIGGER_FL_FILTER` is set?

Reply:
Right, it should not have `NONE` together with `TRIGGER_FL_FILTER`.